coretex/cli/resources/_coretex.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+
+import sys
+import logging
+import runpy
+
+from coretex import _task, TaskRun
+from coretex._task.local.arg_parser import LocalArgumentParser
+from coretex._task.local.task_config import readTaskConfig
+from coretex._task.local.local import LocalTaskCallback
+from coretex.entities import validateParameters
+from coretex.networking import RequestFailedError
+
+
+if __name__ == "__main__":
+    parameters = readTaskConfig()
+
+    parser = LocalArgumentParser(parameters)
+    namespace, _ = parser.parse_known_args(sys.argv)
+
+    taskRunId = namespace.taskRunId
+    refreshToken = namespace.refreshToken
+
+    for parameter in parameters:
+        parameter.value = parser.getParameter(parameter.name, parameter.value)
+
+    parameterValidationResults = validateParameters(parameters, verbose = True)
+    if not all(parameterValidationResults.values()):
+        # Using this to make the parameter errors more readable without scrolling through the console
+        sys.exit(1)
+
+    if not isinstance(taskRunId, int):
+        raise TypeError(f"Invalid type of taskRunId expected \"int\", recieved: \"{type(taskRunId)}\"")
+
+    if not isinstance(taskRunId, int):
+        raise TypeError(f"Invalid type of taskRunId expected \"int\", recieved: \"{type(taskRunId)}\"")
+
+    taskRun: TaskRun = TaskRun.fetchById(taskRunId)
+    callback = LocalTaskCallback(taskRun, refreshToken)
```

Comment:
There is no longer a need to differentiate between Node flow and Local flow. The same _coretex.py script can be used in both cases. Move the script used by Node to coretexpylib and pull it from there for both Node and CLI. Use `processRemote` function in both cases and rename it to be generic as it no longer differentiates between remote and local runs, the flow is the same.